apps/backend/src/mcp/tools/device_management.py

```
@@ -536,7 +541,9 @@ async def get_device_info(
     except Exception as e:
         logger.error(f"Error getting device info for {device}: {e}")
         raise DatabaseOperationError(
-            operation="get_device_info", details={"device": device, "error": str(e)}
+            message=f"Failed to get device info: {str(e)}",
+            operation="get_device_info", 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)